make/autoconf/libraries.m4

```diff
@@ -81,6 +82,13 @@ AC_DEFUN_ONCE([LIB_DETERMINE_DEPENDENCIES],
     NEEDS_LIB_ALSA=false
   fi
 
+  # Check if krb5 is needed
+  if test "x$OPENJDK_TARGET_OS" = xlinux -o "x$OPENJDK_TARGET_OS" = xmacosx; then
+    NEEDS_LIB_KRB5=true
```

Comment:
My understanding is that this external dependency is only needed on Linux. The resulting variable even has linux in the name.